frank-flow/src/frontend/src/app/header/configuration-settings/configuration-settings.component.html

```diff
@@ -0,0 +1,105 @@
+<ngx-smart-modal
+  #configurationSettingsModal
+  class="configurationSettingsModal"
+  customClass="nsm-centered"
+  identifier="configurationSettingsModal"
+>
+  <div class="configurationSettings-container">
+    <h1>Configuration settings</h1>
```

Comment:
It should be "Flow settings", you're not changing anything to the configuration, just to the way it appears in the flow.

I know it's a bit confusing because they are placed in the configuration, but it shouldn't say anything about the configuration itself.

Reply:
This has been changed